pkg/applier/kpt_applier.go

```diff
@@ -77,38 +77,37 @@ type Applier struct {
 	// errs tracks all the errors the applier encounters.
 	// This field is cleared at the start of the `Applier.Apply` method
 	errs status.MultiError
-	// syncing indicates whether the applier is syncing.
-	syncing bool
```

Comment:
oh man was this another bool used for synchronization?

Reply:
yup. and set outside of the lock.